### PR TITLE
Clarified LeaderStatus fields

### PR DESCRIPTION
**Pull request by maansaake**

### CODE REVIEW

**Review by gusbocju — CHANGES_REQUESTED** [review]
Using timestamps based on seconds instead of milliseconds would potentially annihilate the whole idea of sending multiple status updates per second, as they'd all end up looking "the same" due to identical timestamps.

A higher resolution timestamp based on UNIX Epoch (e.g. following the examples of https://currentmillis.com) though, would require some format adaption and thus either a larger timestamp field, or the extension of status messages with an additional data field representing "milliseconds since last epoch".

File: README.md [hunk 1]
```diff
@@ -67,10 +68,10 @@ This message is sent by a car to indicate that following must come to an end. Bo
 This message includes information about a leading vehicle and contains information relevant for a following car to be able to follow it. The LeaderStatus is sent in regular intervals of 125ms and does not expect a response.
 
 ***Fields***
-* uint32_t timestamp       - The time stamp (the time that the message has been sent) of the leading vehicle.
-* float  speed           - Current speed of the leading vehicle.
-* float steeringAngle    - Current steering angle of the leading vehicle.
-* uint8_t distanceTraveled - The distance travelled since the last status update (according the odometer).
+* uint32_t timestamp - The time stamp (the time that the message has been sent) of the leading vehicle, represented as UNIX Epoch time in seconds.
```
Comment (kosara-g) [inline README.md]: Agreed with @gusbocju  we need milliseconds or we can use the RC nanoseconds epoch time and convert to milliseonds

Comment (timjon) [reply README.md]: +1

**Review by timjon — COMMENTED** [review]
Everything looks good except the things already mentioned

File: README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ This repo contains a V2V communication protocol between autonomous RCs with a fo
 3. ##### [License](https://github.com/DIT168-V2V-responsibles/v2v-protocol#3-license)
 4. ##### [Requests](https://github.com/DIT168-V2V-responsibles/v2v-protocol#4-protocol-requests)
 5. ##### [CID ranges](https://github.com/DIT168-V2V-responsibles/v2v-protocol#5-cid-ranges)
+5. ##### [NTP](https://github.com/DIT168-V2V-responsibles/v2v-protocol#6-NTP)
```
Comment (kosara-g) [inline README.md]: here the anchor would probably end in "#6-ntp" in lowercase rather than uppercase so the link might not work as is

**Review by maansaake — COMMENTED** [review]
Updated timestamp field of the LeaderStatus to be a 64 bit unsigned integer. This will now fit milliseconds. Also fixed the rest of the review comments, hopefully no errors remain :-)

**Review by kosara-g — APPROVED** [review]
seems fine to me!

**Review by BigPapaKlonas — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -67,10 +68,10 @@ This message is sent by a car to indicate that following must come to an end. Bo
...
 
 ***Fields***
-* uint32_t timestamp       - The time stamp (the time that the message has been sent) of the leading vehicle.
-* float  speed           - Current speed of the leading vehicle.
-* float steeringAngle    - Current steering angle of the leading vehicle.
-* uint8_t distanceTraveled - The distance travelled since the last status update.
+* uint64_t timestamp - The time stamp (the time that the message has been sent) of the leading vehicle, represented as UNIX Epoch time in milliseconds.
+* float speed - Current speed of the leading vehicle, the leading car should put its current Pedal Position Reading into this message field.
+* float steeringAngle - Current steering angle of the leading vehicle, the leading car should put its current Ground Steering Reading into this message field.
+* uint8_t distanceTraveled - The distance travelled since the last status update (according the odometer), represented in centimeters.
```
Comment (kosara-g) [inline README.md:74]: wait, didn't we say that there was no odometer?

**Comment by maansaake** [comment]
Merging after review changes and approvals

File: README.md (hunk 3 above)
Comment (maansaake) [reply README.md:74]: There is no odometer, distanceTraveled will be whatever the sending car deems fit to put there, be it readings from the IMU or something else

**Pull request merged**